samples/sensor/max31865/prj.conf

```diff
@@ -0,0 +1,11 @@
+CONFIG_PRINTK=y
+CONFIG_CONSOLE=y
+CONFIG_USE_SEGGER_RTT=y
+CONFIG_RTT_CONSOLE=y
+CONFIG_UART_CONSOLE=n
+CONFIG_SHELL=y
+CONFIG_SHELL_BACKEND_RTT=y
+CONFIG_SERIAL=n
```

Comment:
Does this require shell to work properly? If not, please remove `CONFIG_SHELL`.

Also, This looks like a generic sample where any board with a SPI bus can be used. So I think it is better to move the config to enabling RTT and disabling UART to a board overlay (under this sample). This will allow other boards with different serial/console to use this sample.